pkg/apis/crds/karpenter.sh_nodepools.yaml

```diff
@@ -111,6 +112,7 @@ spec:
                         - message: '''schedule'' must be set with ''duration'''
                           rule: self.all(x, has(x.schedule) == has(x.duration))
                     consolidateAfter:
+                      default: 15s
```

Comment:
One option is to keep this as required for WhenEmpty so that we're not picking the default for them, and allow whenUnderutilized to default it to 15s. WDYT?